lib/folio_client.rb

```diff
@@ -120,7 +123,8 @@ def put(path, body = nil, content_type: "application/json")
   def connection
     @connection ||= Faraday.new(
       url: config.url,
-      headers: DEFAULT_HEADERS.merge(config.okapi_headers || {})
+      headers: DEFAULT_HEADERS.merge(config.okapi_headers || {}),
+      request: {timeout: 120}
```

Comment:
I wonder if we should consider making `120` a default value that can be injected by consumers at initialization time?

Reply:
That all makes sense, will take a look at making it a default instead.

Reply:
Added a default_timeout. Thanks for the suggestion, @mjgiarlo and @edsu.